playground/multi-worker/__tests__/with-wrangler-warning/multi-worker.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { describe, expect, test } from 'vitest';
+import { getJsonResponse, isBuild, serverLogs } from '../../../__test-utils__';
+
+describe('multi-worker basic functionality', async () => {
+	test('wrangler warnings are present in the terminal', async () => {
```

Comment:
I think we should only run this test in dev. We shouldn't be validating or warning on the output configuration that we generate.

Reply:
yes, that's my bad, as I mentioned I didn't think of the preview config being generated by us when working on this 👍

Reply:
again I replied too quickly 😅 

Don't you think showing these warnings on `build` is correct?

I agree with you on `preview` since it relies on the generated worker config file, but on `build` we're still dealing with the user's one

Reply:
so I think that line 13 should change:
```diff
-[expectedWarning, expectedWarning],
+[expectedWarning],
```

Also this made me realize that I need to also remove the non-applicable configs from the config file we generate in this PR

Reply:
Yes, you're right. We should warn in build but not preview.

Reply:
yeah I made sure that's what's happening now 🙂👍 (https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/112/commits/9a39b9f2459603d856e81211e12a663f2b615dde)